packages/tools/lib/create-icons/index.js

```diff
@@ -7,23 +7,25 @@ const destDir = `dist/icons/`;
 
 mkdirp.sync(destDir);
 
-const template = (name, pathData) => `import { registerIcon } from "@ui5/webcomponents-base/dist/SVGIconRegistry.js";
+const template = (name, pathData, ltr) => `import { registerIcon } from "@ui5/webcomponents-base/dist/SVGIconRegistry.js";
 
 const name = "${name}";
 const pathData = "${pathData}";
+const ltr = "${ltr}";
 
-registerIcon(name, { pathData });
+registerIcon(name, { pathData, ltr});
 
 export default { pathData };`;
 
-const accTemplate = (name, pathData, accData) => `import { registerIcon } from "@ui5/webcomponents-base/dist/SVGIconRegistry.js";
+const accTemplate = (name, pathData, ltr, accData) => `import { registerIcon } from "@ui5/webcomponents-base/dist/SVGIconRegistry.js";
 import { ${accData.key} } from "../generated/i18n/i18n-defaults.js";
 
 const name = "${name}";
 const pathData = "${pathData}";
+const ltr = "${ltr}";

```

Comment:
This becomes:
```js
const name = "checklist";
const pathData = "M256 2.................";
const ltr = "false";
```
There are "" around false.

Reply:
not needed, removed